templates/ehub.json

```diff
@@ -53,6 +53,34 @@
             "metadata": {
                 "description": "Please use the actual name if using existing event hub."
             }
+        },
+         "newEhubMaxThroughputUnits": {
+            "type": "int",
+            "defaultValue": 20,
+            "metadata": {
+                "description": "The maximum number of throughput units for the Event Hub."
```

Comment:
please mention this is `Optional` parameter.

Reply:
resolved